src/ClientData/include/ClientData/FunctionUtils.h

```diff
@@ -34,7 +34,11 @@ namespace function_utils {
 [[nodiscard]] uint64_t Offset(const orbit_client_protos::FunctionInfo& func,
                               const ModuleData& module);
 
-[[nodiscard]] std::optional<uint64_t> GetAbsoluteAddress(
+// This function should not be used, since it could return incomplete or invalid
+// result in the case when one module is mapped 2 or more times. Try to find another way
+// of solving the problem, for example by converting an absolute address to a module offset and
+// then operating on that.
```

Comment:
I dont know if we have a bug for deprecating the "old" capture format. Iff we do, please add a TODO

Reply:
added todo for disassembly bug - once this is fixed this function can be removed.